docs/install.md

```diff
@@ -50,13 +50,24 @@ $ docker run --rm --name presidio-api --network mynetwork -d -p 8080:8080 -e WEB
 
 4. Verify that Redis and Traefik/NGINX are installed correctly
 
-5. Deploy from `/charts/presidio`
+5. Verify that Kubernetes has access to the [Docker Registry](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-auth-aks).
 
+    - If using a Kubernetes secert, make sure it is registered under 'presidio' namespace
```

Comment:
I think we should elaborate on this, or add a link to documentation. Is this a Presidio requirement or a user requirement?

Reply:
its linked now, and moved to development as well.

Reply:
I personally don't think that the installation page should direct to the readme. Maybe the other way around. the installation should have different installation options, and the readme can direct the user to a specific option or to the installation page in general. WDYT?

Reply:
true, but I wanted to mention the quick installation option, detailed in the readme.
would you rather remove that part from the readme and move it to the installation page?